app/template/app/index.html

```diff
@@ -0,0 +1,49 @@
+{% load static %}
```

Comment:
no need to use this tag, since no static file is used.

Reply:
fixed